src/app/user/Stake/StakeInputNew/StakeInput.tsx

```diff
@@ -1,68 +1,60 @@
-import { ReactNode } from 'react'
-import { Paragraph } from '@/components/TypographyNew'
 import { InputNew } from '@/components/Input'
-import { variantClasses } from '@/components/TypographyNew/Typography'
 import { TokenImage } from '@/components/TokenImage'
+import { Paragraph } from '@/components/TypographyNew'
+import { variantClasses } from '@/components/TypographyNew/Typography'
 import { cn } from '@/lib/utils'
+import Image from 'next/image'
 
 interface Props {
   onChange: (value: string) => void
   value: string
   symbol: string
   labelText?: string
   currencyValue?: string
-  icon?: ReactNode
-  error?: ReactNode
+  errorText?: string
   decimalScale?: number
 }
 
-/**
- * @param onChange
- * @param value
- * @param symbol
- * @param labelText
- * @param currencyValue
- * @param error this can be a string or a ReactNode. It should be accompanied by a red icon per design - out of scope
- * @param decimalScale
- * @constructor
- */
 export const StakeInput = ({
   onChange,
   value,
   symbol,
   labelText,
   currencyValue,
-  error,
+  errorText,
   decimalScale,
 }: Props) => {
   return (
-    <div className="flex flex-col py-[12px] px-[16px] rounded-[4px] w-full bg-bg-60">
-      {labelText && <Paragraph className="mb-[12px]">{labelText}</Paragraph>}
+    <div className="flex flex-col py-3 px-4 rounded-1 w-full bg-bg-60">
+      {labelText && <Paragraph className="mb-3">{labelText}</Paragraph>}
       <div className="flex gap-2">
         <InputNew
           name="amount-stake"
           type="number"
           value={value}
           onChange={onChange}
-          className={cn('grow', variantClasses.h1, error ? 'text-error' : 'text-bg-0')}
+          className={cn('grow', variantClasses.h1, errorText ? 'text-error' : 'text-bg-0')}
           placeholder="0"
           inputProps={{ decimalScale }}
         />
-        <div className="flex flex-row gap-x-[4px] items-center">
+        <div className="flex items-center gap-1 shrink-0">
           <TokenImage symbol={symbol} size={24} />
           <Paragraph variant="body-l" bold>
             {symbol}
           </Paragraph>
         </div>
       </div>
       {currencyValue && (
-        <div>
-          <Paragraph variant="body-s" bold className="text-bg-0">
-            {currencyValue}
-          </Paragraph>
+        <Paragraph variant="body-s" className="text-bg-0">
+          {currencyValue}
+        </Paragraph>
+      )}
+      {errorText && (
+        <div className="flex items-center gap-2 mt-2">
```

Comment:
Can you extract this to a separate component? 
I am sure we will be using this somewhere else